### PR TITLE
Add support for `FinishedAt`

### PR DESCRIPTION
**Pull request by euank**
The unit tests pass.

I also did the following manual testing:

```
$ cat ./examples/sleep/sleepy.yaml 
apiVersion: v1
kind: Pod
metadata:
  labels:
    name: sleepy
  name: sleepy
spec:
  restartPolicy: Never
  containers:
    - name: master
      image: busybox
      resources:
        limits:
          cpu: "0.1"
      command: ["sh", "-c", "sleep 30"]
$ kubectl create -f ./examples/sleep/sleepy.yaml
$ # wait
$ kubectl get pod sleepy -o yaml | grep -E "Time:|At:" 
  - lastProbeTime: null
    lastTransitionTime: 2016-04-15T21:50:32Z
        finishedAt: 2016-04-15T21:50:32Z
        startedAt: 2016-04-15T21:50:01Z
  startTime: 2016-04-15T21:49:58Z
```

cc @yifan-gu @sjpotter

### CODE REVIEW

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
 	"ImportPath": "k8s.io/kubernetes",
 	"GoVersion": "go1.5",
-	"GodepVersion": "v60",
```
Comment (yifan-gu) [inline Godeps/Godeps.json]: Please keep this as a doc.

File: pkg/kubelet/rkt/rkt.go [hunk 2]
```diff
@@ -82,7 +82,6 @@ const (
 	k8sRktNameAnno         = "io.kubernetes.pod.name"
 	k8sRktNamespaceAnno    = "io.kubernetes.pod.namespace"
 	//TODO: remove the creation time annotation once this is closed: https://github.com/coreos/rkt/issues/1789
-	k8sRktCreationTimeAnno           = "io.kubernetes.container.created"
```
Comment (yifan-gu) [inline pkg/kubelet/rkt/rkt.go]: @euank Remove the TODO above this line as well?

File: pkg/kubelet/rkt/rkt.go [hunk 3]
```diff
@@ -1569,6 +1557,8 @@ func populateContainerStatus(pod rktapi.Pod, app rktapi.App, runtimeApp appcsche
 	}, nil
 }
 
+// GetPodStatus returns the cached status for the provided pod UID, as well as
+// whether it was a cache hit.
```
Comment (yifan-gu) [inline pkg/kubelet/rkt/rkt.go]: comments are not necessary true.

Comment (euank) [inline pkg/kubelet/rkt/rkt.go]: Yup, bad comment copied from elsewhere, I'll fix it.

**Comment by yifan-gu** [comment]
LGTM, just some nits.

**Comment by yifan-gu** [comment]
LGTM

**Pull request merged**